### ISSUE TITLE
Support setting revs_limit via REST in LiteServ

### ISSUE THREAD

**Issue by sethrosetter** [couchbase/couchbase-lite-android#905]

**Comment by hideki** [comment]
@sethrosetter Does this block your auto-mate testing?

**Comment by sethrosetter** [comment]
No. However, now would be a good time to discuss matching the rest options that liteserv-ios has and making sure Xamarin.iOS and Xamarin.Android liteserv are consistent as well.

**Comment by hideki** [comment]
@sethrosetter Tomorrow's all platforms sprint planing is good time to discuss this topic.
